.github/CONTRIBUTING.md

```diff
@@ -202,6 +238,7 @@ There's no indication of what "1" and "2" mean! Instead, you'd do something like
 			(...)
 ````
 This is clearer and enhances readability of your code! Get used to doing it!
+On another note, if the define is only used in one place only with a clearly named variable (var/cost), then adding a define would further obfuscate it, use the regular data types in those cases.
```

Comment:
What does this mean?

Reply:
it's talking about xeno abilities where instead of the cooldown var being defined as 12 SECONDS it's set to ABILITY_NAME_COOLDOWN which is a define for 12 SECONDS

Reply:
I've done that before when I want a defines file to act as a source for balance regarding a feature. It makes sense to do that depending on context

Reply:
yeah its not something you can make a blanket rule for its a case by case thing

Reply:
Removed.